Core.Telemetry/OpenTelemetryLoggerProvider.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public class OpenTelemetryLoggerProvider : ILoggerProvider
```

Comment:
>OpenTelemetryLoggerProvider [](http://example.com/codeflow?start=17&length=27)

Just wondering: did we get these from a Wiki or documentation?

Reply:
Yes, sample code before modification was found here: https://eng.ms/docs/products/geneva/collect/instrument/opentelemetrydotnet/otel-logs

Reply:
Can you please add this link here as well?